modules/text_generation.py

```diff
@@ -89,8 +97,12 @@ def _generate_reply(question, state, stopping_strings=None, is_chat=False):
     yield reply
 
 
+encode_llama_prompts = ['LlamaCppModel', 'RWKVModel', 'CtransformersModel']
```

Comment:
I don't like these variables floating here and think that they should be removed.

Reply:
I hate probing by class names, but I have put them back for now. I'll try to refactor them next time.